src/plugin.ts

```diff
@@ -135,14 +137,12 @@ function parseResource(
  * @example
  *  parseMessage("test", "Hello world")
  */
-function parseMessage(id: string, value: string): ast.Message {
-
-  const regex = /({[^}]+})/g;
-  const splitArray = value.split(regex);
+function parseMessage(id: string, value: string, regex?: string): ast.Message {
 
+  const splitArray = value.split(regex || '(?!.*)');
```

Comment:
why don't you check if regex is undefined and if regex is undefined don't apply placeholder logic?